site/app/libraries/Access.php

```diff
@@ -414,7 +414,9 @@ public function canUser($user, string $action, $args) {
                         !($group === User::GROUP_FULL_ACCESS_GRADER && !$gradeable->isTaGrading())
                         &&
                         //Students are allowed to see this if its a peer graded assignment
-                        !($group === User::GROUP_STUDENT && $gradeable->isPeerGrading())
+                        !($group === User::GROUP_STUDENT && $gradeable->isPeerGrading() && $gradeable->getGradeStartDate() <= $this->core->getDateTimeNow())
+                        &&
```

Comment:
why is this check getting and'd twice in the same if statement `$gradeable->getGradeStartDate() <= $this->core->getDateTimeNow())` ?

Reply:
Good point, rewrote to only check once